test/sdk/restapi/alpha/swagger/git_push.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
🤔 what's this for? Generated as well?

Reply:
This is generated but un-necessary. Removed